inc/saitypes.h

```diff
@@ -280,8 +276,7 @@ typedef enum _sai_object_type_t
     SAI_OBJECT_TYPE_MACSEC_SC                = 91,
     SAI_OBJECT_TYPE_MACSEC_SA                = 92,
 
-    // Do not change position of SAI_OBJECT_TYPE_MAX
-    SAI_OBJECT_TYPE_MAX                      = 92,
+    SAI_OBJECT_TYPE_MAX,  /* Must remain in last position */
```

Comment:
add explicit number 93 as it was before ?

Reply:
Thought it might be less edits this way when new enums are added.  Is there an advantage to explicit definition for the MAX?

Reply:
no code advantage, just code style look

Reply:
I believe in this style we don't have to bump SAI_OBJECT_TYPE_MAX every time a new entry is added. If you feel strongly about the keeping the old cold style just respond and I will change this back.

Reply:
ok, let it be as is